### PR TITLE
[1.x] Adds tests against "openswoole"

### PR DESCRIPTION
**Pull request by nunomaduro**
This pull request duplicates the tests workflow, so we can test octane against "openswoole". A good news, seems that "open-swoole" is already compatible with PHP 8.1. Yet, "swoole" is not, so we are going to ignore the PHP 8.1 swoole build for now.

### CODE REVIEW

**Comment by driesvints** [comment]
@nunomaduro I'd personally keep this grouped under a single file as a different "job". The way we usually organise GHA workflows is to group specific "type" of workflows. 

Like framework: types, tests, database engines.

Swoole and OpenSwoole tests basically test the actual working of Octane so it's best to keep these grouped under "tests.yml". All OS projects of Laravel also have a "tests.yml" file.

**Comment by nunomaduro** [comment]
Checking Dries...

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  tests:
+  tests / swoole:
```
Comment (driesvints) [inline .github/workflows/tests.yml:10]:
```suggestion
  swoole:
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -37,3 +37,34 @@ jobs:
 
       - name: Execute tests
         run: vendor/bin/phpunit --verbose
+        continue-on-error: ${{ matrix.php > 8 }}
+  tests / openswoole:
```
Comment (driesvints) [inline .github/workflows/tests.yml]:
```suggestion
  openswoole:
```

**Pull request merged**